subprojects/gradle/runner/shared/src/main/kotlin/com/avito/runner/CommandLineExecutor.kt

```diff
@@ -67,15 +67,18 @@ interface CommandLineExecutor {
 
                 reader.close()
 
-                val exitCode = process.waitFor()
-
-                when (exitCode) {
+                when (val exitCode = process.waitFor()) {
                     0 -> {
                         emitter.onNext(ProcessNotification.Exit(buffer.toString()))
                         emitter.onCompleted()
                     }
                     else -> {
-                        emitter.onError(IllegalStateException("Process $commandAndArgs exited with non-zero code $exitCode"))
+                        emitter.onError(
+                            IllegalStateException(
+                                "Process $commandAndArgs exited with non-zero code $exitCode. " +
+                                    "Output: $buffer"
```

Comment:
We were loosing an original stdout/stderr from adb command.

Before:
```
... exited with non-zero code 1
```

Now:
```
... exited with non-zero code 1. Output: adb: device 'emulator-5554' not found
```